src/main/frontend/modules/shortcut/dicts.cljc

```diff
@@ -1,12 +1,10 @@
 (ns ^:bb-compatible frontend.modules.shortcut.dicts
   "Provides dictionary entries for shortcuts"
   (:require [medley.core :as medley]))
-
```

Comment:
Could you remove all the whitespace removals? This makes the code harder to read